pkg/acsengine/const.go

```diff
@@ -90,6 +102,12 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"dnsmasq":      "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":        "pause-amd64:3.0",
 		"windowszip":   "v1.6.0intwinnat.zip",
+		"nodestatusfreq":   "10s",
+		"nodegraceperiod":   "40s",
+		"podeviction":   "5m0s",
+		"routeperiod":   "10s",
```

Comment:
These aren't images and none of them vary by k8s version. Could we move these default consts to be siblings with the KubeImages map?  We can still override in the future when we allow the user to specifiy in the UX.